utils/src/test/scala/za/co/absa/enceladus/utils/types/TypedStructFieldSuite.scala

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
This has been added as part of #908.
Can you exclude it please, to avoid merge conflict/need for merge commit?

Reply:
How about we merge yours first and I'll rebase this branch? I'm not scared of merge conflicts.

Reply:
I thought you would be changing the `xml` failing to notice, it's already specific.